no-computer.md

```diff
@@ -1,55 +1,68 @@
 # No computer project
-
+Baseball! It's TOTALLY a DSL.
 
 ## The user and a language
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+This language would be good for people who want to run baseball simulations to ask "what if" questions. If a pitcher throws the ball a certain way to a certain location to a certain hitter who swings a certain way, what will happen?
```

Comment:
I think this exists as a board game my dad loves and played originally as a kid. Have you ever heard of [Strat-O-Matic Baseball](http://www.strat-o-matic.com/)? There is even have a computer game version my dad had on a bunch of floppy discs when I was a kid.